services/search-api/lambdas/searchBookings.js

```diff
@@ -0,0 +1,18 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import { searchBookings } from '../helpers/search';
+
+export async function main(event) {
+  const body = { ...JSON.parse(event.body) };
+
+  const [requestError, getSearchResponse] = await to(searchBookings(body));
+  if (requestError) {
+    throwError(requestError.status, requestError.errorMessage);
```

Comment:
I think that the lambda should return an error response instead of throwing. Check **response.failure** function on other lambdas